.travis.yml

```
@@ -7,3 +7,4 @@ install:
 
 script:
   - ansible-playbook -i herp, bastion.yml install-ci.yml provision.yml --syntax-check
+  - python test/layout-checks.py job-list.txt
```

So we've changed this out from under you. Instead of another line in the script, it needs to be another entry in the `env` section, so that we can get them to run in parallel (and fail early).